petri-amplitude-testapp/src/main/webapp/WEB-INF/laboratory.properties

```diff
@@ -1,11 +1,11 @@
 petri.url=http://localhost:9010/petri
 amplitude.url=https://api.amplitude.com/httpapi
 amplitude.api.key=198e3469868de498f5d67581d6de4518
-amplitude.timeout.ms=500
+amplitude.timeout.ms=1000
 
 petri.specs.pkg=specs
 reporter.interval = 1
-petri.writeStateToServer = false
+petri.writeStateToServer = true
```

Comment:
did anything make you change this?
it should not be needed (since you use cookie for userid)